pom.xml

```diff
@@ -1,48 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  - Copyright 2024-present Coinbase Global, Inc.
+  -
+  -  Licensed under the Apache License, Version 2.0 (the "License");
+  -  you may not use this file except in compliance with the License.
+  -  You may obtain a copy of the License at
+  -
+  -  http://www.apache.org/licenses/LICENSE-2.0
+  -
+  -  Unless required by applicable law or agreed to in writing, software
+  -  distributed under the License is distributed on an "AS IS" BASIS,
+  -  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  -  See the License for the specific language governing permissions and
+  -  limitations under the License.
+  -->
+
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 
-    <groupId>com.coinbase</groupId>
-    <artifactId>intx-java-sdk</artifactId>
-    <version>0.1.0</version>
+		<modelVersion>4.0.0</modelVersion>
+    <artifactId>coinbase-intx-sdk-java</artifactId>
+		<groupId>com.coinbase.intx</groupId>
+		<name>Coinbase International Exchange Java SDK</name>
+		<description>Sample Java SDK for the Coinbase International Exchange REST APIs</description>
+		<version>0.1.1</version>
+		<url>https://github.com/coinbase-samples/intx-sdk-java</url>
 
     <licenses>
         <license>
             <name>Apache License, Version 2.0</name>
-            <url>https://www.apache.org/licenses/LICENSE-2.0</url>
+						<url>https://github.com/coinbase-samples/intx-sdk-java/blob/main/LICENSE</url>
```

Comment:
looks like some tab vs spaces issue here

Reply:
Agreed. It renders ok in VIM, but I will address in a subsequent PR. We need these approved before the STP meeting tomorrow.

Reply:
Actually, I fixed